content/js/record.md

```diff
@@ -323,13 +347,18 @@ skygear.publicDB.query(new skygear.Query(Note))
 }
 ```
 
-Query on reserved columns example:
+Querying reserved columns:
 
 ``` javascript
-let query = new skygear.Query(Note);
-query.equalTo('_owner', skygear.currentUser.id);
-// '_owner' is an alias for '_owner_id'
-skygear.publicDB.query(query);
+const myNotesQuery = new skygear.Query(Note)
+  .equalTo('_owner_id', skygear.currentUser.id);
+
+skygear.publicDB.query(myNotesQuery)
+  .then((results) => {
+    console.log(results);
+    // [ ... ]
```

Comment:
you may say that the `results` are all records that is created (as owner) by the current user